merge_requests.go

```diff
@@ -106,6 +106,7 @@ type MergeRequest struct {
 	HasConflicts                bool                   `json:"has_conflicts"`
 	BlockingDiscussionsResolved bool                   `json:"blocking_discussions_resolved"`
 	Overflow                    bool                   `json:"overflow"`
+	Draft                       bool                   `json:"draft"`
```

Comment:
Please place this between `Description` and `WorkInProgress` so it's inline with the ordering in the docs (as all fields are ordered following the docs).

Reply:
done!